### PR TITLE
[ETFE-3084] prepare frontend for public beta

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: app/controllers/ConfirmationController.scala [hunk 1]
```diff
@@ -42,9 +42,11 @@ class ConfirmationController @Inject()(
 
   def onPageLoad(ern: String, arc: String): Action[AnyContent] =
     authorisedDataRequestWithCachedMovementAsync(ern, arc) { implicit request =>
-      withAnswer(ConfirmationPage) { confirmationDetails =>
-        logger.info("[onPageLoad] Successful Explain Delay confirmation page displayed")
-        Future.successful(Ok(view(confirmationDetails)))
+      withAnswer(ConfirmationPage) { _ =>
```
Comment (machinesleet) [inline app/controllers/ConfirmationController.scala:45]: what as the reason for this change?

Comment (james-work-account) [reply app/controllers/ConfirmationController.scala:45]: `confirmationDetails` isn't used anywhere so I changed it to `_`, however it doesn't hurt to keep the check to make sure it's there.

Comment (machinesleet) [reply app/controllers/ConfirmationController.scala:45]: Sorry I meant this whole change in general

Comment (james-work-account) [reply app/controllers/ConfirmationController.scala:45]: I'm going to make custom visualisations in Kibana to track how many of the various cancellation reasons we have, similar to what we've already got for RoR and explain delay. I need a custom log for this.

**Review by machinesleet — APPROVED** [review]

**Pull request merged**